modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Layout/Header/HeaderTabs.tsx

```diff
@@ -14,30 +14,35 @@
 
 import ClayTabs from '@clayui/tabs';
 import {useAtom} from 'jotai';
-import {useNavigate} from 'react-router-dom';
+import {useLocation, useNavigate} from 'react-router-dom';
 import {headerAtom} from '~/atoms';
 
 const HeaderTabs = () => {
 	const [tabs] = useAtom(headerAtom.tabs);
 
+	const {pathname} = useLocation();
+	const isEditCase = pathname.includes('edit' || 'complete');
+
 	const navigate = useNavigate();
 
-	return (
-		<ClayTabs className="tr-header-container__tabs">
-			{tabs.map((tab, index) => (
-				<ClayTabs.Item
-					active={tab.active}
-					innerProps={{
-						'aria-controls': `tabpanel-${index}`,
-					}}
-					key={index}
-					onClick={() => navigate(tab.path)}
-				>
-					{tab.title}
-				</ClayTabs.Item>
-			))}
-		</ClayTabs>
-	);
+	if (!isEditCase) {
+		return (
+			<ClayTabs className="tr-header-container__tabs">
+				{tabs.map((tab, index) => (
+					<ClayTabs.Item
+						active={tab.active}
+						innerProps={{
+							'aria-controls': `tabpanel-${index}`,
+						}}
+						key={index}
+						onClick={() => navigate(tab.path)}
+					>
+						{tab.title}
+					</ClayTabs.Item>
+				))}
+			</ClayTabs>
+		);
+	}
 };
 
 export default HeaderTabs;
```

Comment:
This change shouldn't be made in this component, because this is a global component and this component is used for some pages in different contexts, you can make this change in the outlet of the component (CaseResultOutlet), in Testray we have a hook (useHeader) to control the display of header, this hook is used in this outlet but no has validation to remove it when you edit something.

Reply:
done!